tests/virt_autotest/libvirt_host_bridge_virtual_network.pm

```diff
@@ -17,15 +17,15 @@ use strict;
 use warnings;
 use testapi;
 use utils;
-use version_utils qw(is_sle is_alp);
+use version_utils qw(is_sle is_alp is_tumbleweed);
 
 our $virt_host_bridge = 'br0';
 our $based_guest_dir = 'tmp';
 sub run_test {
     my ($self) = @_;
 
     # ALP has done this in earlier setup
-    unless (is_alp) {
+    unless (is_alp || is_tumbleweed) {
```

Comment:
Is it not required? I don't know much about this network tests, but TW should be more like SLE than ALP. Please confirm with Leon.

Reply:
Make sence. Thanks for the explanation.